module/os/linux/zfs/zpl_file.c

```diff
@@ -620,6 +620,18 @@ zpl_readpage(struct file *filp, struct page *pp)
 	return (error);
 }
 
+static int
+zpl_readpage(struct file *filp, struct page *pp)
+{
+	return (zpl_readpage_common(pp));
+}
+
+static int
+zpl_readpage_filler(void *data, struct page *page)
```

Comment:
nit: let's use `pp` for the page variable just for consistency with `zpl_readpage()`.